opengrok-indexer/src/main/java/org/opengrok/indexer/web/EftarFile.java

```diff
@@ -87,56 +81,6 @@ public Node get(long hash) {
         }
     }
 
-    class FNode {
```

Comment:
?

Reply:
It was not used at all -> YAGNI. If we need it is needed again we can add it back.

Reply:
ok